Random crashes all over the app (IOS/Android)

Unfortunately there are no clear steps yet how to reproduce crash

Here is [logcat](https://drive.google.com/file/d/1sPoQXsDjETmBtiQF8JZJgPWxKZfK9BGh/view?usp=share_link) from Android device. It should contain logs of 1 crash. 

### Additional Information

- Status version: release 1.22 
- Operating System: Android, iOS


[comment]: # (Please, add logs/notes if necessary)

I looked at the prev shared logcat and saw two types of panic:
```
02-13 15:14:59.914  3140  3239 E GoLog   : panic: runtime error: slice bounds out of range [:2] with capacity 0
```
```
panic: enode: can't sign record: record bigger than 300 bytes
```

The first one SHOULD be fixed by the PR I created yesterday for status-go, but the second one, I'm not sure how to fix it.
The problem goes like this: The function `writeMultiaddressField` is called when there's an update in the multiaddresses for the node, and one of the multiaddresses contain a circuit-relay addr.

It attempts to write as many multiaddresses as possible in the multiaddr field, but since  go-ethereum will not complain until the moment `.Node()` is executed, and if it exceeds 300 bytes, it will panic. I try to intentionally cause the panic here:
https://github.com/status-im/status-go/blob/8acc46f758729b44d771b5713553c2c42109b548/vendor/github.com/waku-org/go-waku/waku/v2/node/localnode.go#L51 and capture it here: https://github.com/status-im/status-go/blob/8acc46f758729b44d771b5713553c2c42109b548/vendor/github.com/waku-org/go-waku/waku/v2/node/localnode.go#L30-L34 so instead of panicing and crashing, it returns an error instead. But as we can see, this is not effective....

I'm thinking of just removing storing circuit relay addresses in the ENR. What do you think? I dont see anything obvious that could be causing the error in the code, and it seems to happen rarely

--- a/go.mod
+++ b/go.mod
@@ -78,7 +78,7 @@ require (
 	github.com/ipfs/go-log/v2 v2.5.1
 	github.com/ladydascalie/currency v1.6.0
 	github.com/meirf/gopart v0.0.0-20180520194036-37e9492a85a8
-	github.com/waku-org/go-waku v0.5.2-0.20230221221637-74b734c337e9
+	github.com/waku-org/go-waku v0.5.2-0.20230223234511-57f6110074d0
 	github.com/yeqown/go-qrcode/v2 v2.2.1
 	github.com/yeqown/go-qrcode/writer/standard v1.2.1
 )

